src/pe_reports/charts.py

```diff
@@ -0,0 +1,162 @@
+"""Class methods for report charts."""
+
+# Third-Party Libraries
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MaxNLocator
+
+
+class Charts:
+    """Build charts."""
+
+    def __init__(self, df, width, height, name, title, x_label, y_label):
+        """Initialize chart class."""
+        self.df = df
+        self.title = title
+        self.x_label = x_label
+        self.y_label = y_label
+        self.width = width
+        self.height = height
+        self.name = name
+
+    def pie(self):
+        """Build pie chart."""
+        df = self.df
+        width = self.width
+        height = self.height
+        name = self.name
+        plt.rcParams.update({"figure.max_open_warning": 0})
+        catName = df.columns[0]
+        Val_1Name = df.columns[1]
+        df = df.sort_values(by=Val_1Name, ascending=False)
+        Category_column = df[catName]
+        Value_column = df[df.columns[1]]
+        labels = Category_column
+        plt.gca().axis("equal")
+
+        # Only show the label when the value is greater than 10%.
+        def autopct(pct):
+            """Get autopct."""
```

Comment:
This docstring should be improved - it's not particularly useful.

Reply:
@dav3r Addressed in this [commit](https://github.com/cisagov/pe-reports/pull/106/commits/874c4c1d018162acd7be4458e78a7d90062657ea).